src/common/http_header.cc

```diff
@@ -50,7 +50,10 @@ const char* encodingString(Encoding encoding) {
         return "chunked";
     case Encoding::Unknown:
     	return "unknown";
+    default:
```

Comment:
Please do not add a "default" for switch statements that switch on enums.  Add the actual missing values.  The compiler warning about missing values from the switch statement is useful, and adding a default just masks the check.